whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageDequeue.java

```diff
@@ -188,15 +191,19 @@ private void handleMessage(final String messageId) {
         final MimeMessage message = mailMessageDao.getMessage(messageId);
 
         try {
+            LOGGER.info("checking bounce message?{}", messageId);
+            if (bouncedMessageService.isBouncedMessage(message)){
```

Comment:
`is()` method call has a not obvious side-effect (i.e. mark email as undeliverable). Should we pull that behaviour out into the `if()` body?